examples/enc_test.go

```diff
@@ -6,26 +6,26 @@ import (
 	"github.com/dedis/crypto/util/random"
 )
 
-func ElGamalEncrypt(suite crypto.Suite, pubkey crypto.Point, message []byte) (
+func ElGamalEncrypt(group crypto.Group, pubkey crypto.Point, message []byte) (
```

Comment:
Shouldn't this be a `Suite` now? Even though if it only contains the `crypto.Group`, this would `ensuring that each package explicitly documents (via that package-specific Suite interface) what precise crypto functionality it depends on.`